src/ontology/mondo-edit.obo

```diff
@@ -410566,6 +410565,7 @@ xref: MedDRA:10036813 {source="Orphanet:683/e", source="Orphanet:683"}
 xref: MESH:D013494 {source="Orphanet:683/e", source="DOID:678", source="MONDO:equivalentTo", source="Orphanet:683"}
 xref: NCIT:C85028 {source="DOID:678", source="MONDO:equivalentTo"}
 xref: NORD:1619 {source="MONDO:NORD"}
+xref: OMIMPS:601104 {source="MONDO:equivalentTo"}
```

Comment:
Similar question about xref since it seems to be a subtype, e.g. "SUPRANUCLEAR PALSY, PROGRESSIVE, 1; PSNP1", but others like `DOID:678` are not "progressive supranuclear palsy".

Reply:
I don't think I understand your comment. 

- OMIM:601104 = the specific subtype (SUPRANUCLEAR PALSY, PROGRESSIVE, 1; PSNP1), 
- OMIMPS:601104 = the general term, not specific to a specific gene.

Therefore this general term gets the OMIMPS id, and the more specific term (MONDO:0010997) gets the OMIM id.

Reply:
Ah, ok - difference between OMIM and OMIMPS reference. Thank you